.editorconfig

```diff
@@ -0,0 +1,11 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 4
+charset = utf-8
+max_line_length = 80
+
+[.travis.yml]
+indent_style = space
+indent_size = 2
```

Comment:
Since we are using [Python in the functional test](https://github.com/spoqa/nirum/tree/0.3-maintenance/test/python), How about adding configuration for Python as well?